src/Core/Domain/Header/Header.php

```diff
@@ -59,7 +59,8 @@ public function __construct(
             $translationDomain = $translator->getDefaultTranslationDomain();
             $defaults['default_translation_domain'] = $translationDomain->getDomain();
             $fallbackDomain = $translationDomain->getFallback()?->getDomain();
-            $defaults['default_translation_domain_fallback'] = $fallbackDomain ?? $defaults['default_translation_domain'];
+            $defaults['default_translation_domain_fallback'] = $fallbackDomain ??
+                $defaults['default_translation_domain'];
```

Comment:
The ?? should be at the start of the line in cases like this

Reply:
fixed 4e1261a65b9c1f3a82a284d86ab28b012142cc16